ui/submodules/packages/components/src/components/componentTable.tsx

```diff
@@ -47,6 +49,21 @@ export function ComponentTable(props: TableViewData<ComponentType>): JSX.Element
                 sortComparator: (v1: string[], v2: string[]) => v1.length - v2.length,
                 renderCell: (props) => <CategoriesCell {...props} filterValue={getFilter("CATEGORY", true)} />,
             },
+            {
+                field: "links",
+                type: "actions",
+                getActions: ({ row }) =>
```

Comment:
getLinks?

Reply:
wtf??

Reply:
@lciolecki `getActions` is a method of a grid, not from our code ;)

Reply:
Ou.. ;)